packages/api/src/graphql/blocks.ts

```diff
@@ -442,8 +442,18 @@ export const GraphQLLinkPageBreakBlock = new GraphQLObjectType<LinkPageBreakBloc
   name: 'LinkPageBreakBlock',
   fields: {
     text: {type: GraphQLString},
+    htmlText: {type: GraphQLString},
     linkURL: {type: GraphQLString},
-    linkText: {type: GraphQLString}
+    linkText: {type: GraphQLString},
+    linkTarget: {type: GraphQLString},
```

Comment:
Either make it boolean for open in same page or new page. Or do it properly and only accept one of the four options.
[_blank, _self, _parent, _top]

Reply:
Good point, not at the moment ([_blank, _self, _parent, _top]), but the idea is to keep this as string to be flexible if we need some other values here, not 100% sure yet which values will be stored here